packages/Breadcrumb/src/Breadcrumb.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+
+import cx from 'classnames';
+
+import ChevronLeft from '@igloo-ui/icons/dist/ChevronLeft';
+import ChevronRight from '@igloo-ui/icons/dist/ChevronRight';
+
+import './breadcrumb.scss';
+
+type Item = {
+  /** Add text for the breadcrumb */
+  label: React.ReactNode;
+  link?: string;
+};
+
+export interface BreadcrumbProps extends React.ComponentProps<'ul'> {
```

Comment:
small detail but we use in the return a `ol`

Reply:
oh shoot good catch!. I was originally going to use a ul and forgot to update it :/